src/components/team/transfer-ownership-confirm-dialog.tsx

```diff
@@ -28,7 +28,7 @@ const TransferOwnershipConfirmDialog: React.FunctionComponent<DialogProps> = ({
       <DialogOverlay isOpen={isOpen} onDismiss={onClose}>
         <DialogContent
           aria-label="Confirm removing team member"
-          className="bg-white rounded-md border-gray-400 p-8"
+          className="dark:bg-gray-900 bg-white rounded-md border-gray-400 p-8 w-96 lg:w-1/3 "
```

Comment:
Generally recommended to keep line width to 45-75 chars. Current width on desktop is ~78char this brings it down to 57 char. Similarly on mobile we were at 22 char width, now its 44